src/Xablu.WebApiClient/Services/Rest/RefitService.cs

```diff
@@ -77,7 +77,7 @@ protected virtual void DisableAutoRedirects(HttpMessageHandler messageHandler)
                 {
                     Logger.Trace("Disabling AutoRedirects");
                 }
-                internalClientHandler.AllowAutoRedirect = false;
+                internalClientHandler.AllowAutoRedirect = true;
```

Comment:
Is this change intended? I didn't see anything about it in the PR description, also the method where this line is located is called `DisableAutoRedirects` 🤔